base/log.go

```diff
@@ -11,10 +11,22 @@ import (
 	reporef "github.com/qri-io/qri/repo/ref"
 )
 
+type DatasetLogItem struct {
+	// Decription of a dataset reference
+	VersionInfo dsref.VersionInfo `json:"versionInfo,omitempty"`
+	// Timestamp field from the commit
+	CommitTime time.Time `json:"commitTime,omitempty"`
```

Comment:
The CommitTime is also extracted as it flows into the DataLogItem better than in VersionInfo due to it's dependency on the dataset.

Reply:
Moving CommitTime back as even though it flows from the same source, it's more relevant and used through the VersionInfo